### DISCUSSION TITLE
new-blog-techspec/

### DISCUSSION THREAD

**Discussion by giscus** [JungWinter/blog#45] [Comments]
# new-blog-techspec/

처음 블로그를 만들고 오랜 시간이 지났습니다.
그동안 블로그를 갈아 엎고 싶은 생각은 항상 있었지만 '왜 그런 마음이 있을까?', '블로그로 무엇을 얻고자 하는거지?' 고민만 하다 '중요하지만 지금 당장 급한건 아니야' 라는 핑계로 지금까지 왔습니다…

https://www.winterjung.dev/new-blog-techspec/

**Comment by kyh196201** [comment]
잘 보고 갑니다 감사합니다!
